cms/templates/partials/hero.html

```diff
@@ -14,19 +14,50 @@ <h1>{{ page.title }}</h1>
           <h2>{{ page.subhead }}</h2>
         </div>
         <div class="mt-5 mb-5">
-          {% if not user.is_anonymous %}
-            {% if not enrolled and product_id %}
-              <a class="enroll-button" href="{% url "checkout-page" %}?product={{ product_id }}">
-                Enroll Now
-              </a>
-            {% elif enrolled %}
-              <a class="enroll-button" href="{% url "user-dashboard" %}">
-                Enrolled
-              </a>
-            {% endif %}
-          {% endif %}
           {% if action_title and action_url %}
             <a id="actionButton" class="enroll-button" href="{{ action_url }}">{{ action_title }}</a>
+          {% else %}
+            {% if not user.is_anonymous %}
+              {% if not enrolled and product_id %}
+                <a class="enroll-button" href="{% url "checkout-page" %}?product={{ product_id }}">
+                  Enroll Now
+                </a>
+              {% elif enrolled %}
+                <a class="enroll-button" href="{% url "user-dashboard" %}">
+                  Enrolled
+                </a>
+              {% endif %}
+            {% else %}
+              <div class="enroll-dropdown">
+                <a class="enroll-button dropdown-toggle" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="true">
+                  Enroll Now
+                </a>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                  <div class="login-popup">
+                    <div class="triangle"></div>
+                    <h4>
+                      <span>Sign In/Create Account</span>
+                      <a class="close-btn">
+                        ×
+                      </a>
+                    </h4>
+                    <div class="popup-text">
+                      Please Sign In to MITx PRO to enroll in a course
+                    </div>
+                    <div class="bottom-row">
+                      <div class="popup-buttons">
+                        <a class="sign-in-link link-button" href="/login">
```

Comment:
The login and signup URL's changed recently, now it's `/signin` and `/create-account`